README.md

```diff
@@ -353,9 +353,9 @@ Create and start a new Linode.
 
 **-l**, **--label**: Required. A Linode to operate on.
 
-**-L**, **--location**: Required. The datacenter to use for deployment. Locations are Atlanta, Dallas, Fremont, London, Newark, and Tokyo.
+**-L**, **--location**, **--datacenter**: Required. The datacenter to use for deployment. Locations are Atlanta, Dallas, Fremont, London, Newark, and Tokyo.
 
-**-d**, **--distribution**: Required when not using imageid. Distribution name or DistributionID to deploy.
+**-d**, **--distribution**, **distro**: Required when not using imageid. Distribution name or DistributionID to deploy.
```

Comment:
For the `linode create` command, only --distribution and -d will resolve. https://github.com/linode/cli/blob/master/lib/Linode/CLI/Util.pm#L66
The `linode distributions` command does have a distros alias, but that isn't this one.